enclave_wrangler/main.py

```diff
@@ -155,12 +162,14 @@ def run(input_csv_folder_path):
     # II. call the REST APIs to create them on the Enclave
     # ...now that we have all the data from concept set are created
     # problem with 16(3623) and 39(5104) 58(1820) 73(6791) 74(1501)
+    temp_testing_cset_id = 1000000273  # Stephanie said this was a draft or archived set - Joe 2022/03/15
     for premade_codeset_id in premade_codeset_ids:
-        #    if premade_codeset_id != 1000000058:
-        #        continue
+        # TODO: temporary debug code to look for missing concept container not showing in the UI
+        # TODO: debug code for adding expressionItems to missing container from UI, l162,l163
+        if premade_codeset_id != temp_testing_cset_id:
```

Comment:
@stephanieshong I left these two lines here just. So right now it's only uploading a single codeset. I think we should probably change it back now?

@Sigfried Stephanie had this here where it is only uploading a single set which we know we're OK to modify (I think it is a draft or an archive set), just to help with development / testing purposes. But if we're actually going to use the `enclave_wrangler`, we'll need to comment out / remove these two lines.

Reply:
Thank you. Once we are done with trouble shooting with Lisa we can remove them. But until then I will leave them here.